src/gentropy/dataset/l2g_prediction.py

```diff
@@ -120,11 +120,16 @@ def to_disease_target_evidence(
             l2g_threshold (float): Threshold to consider a gene as a target. Defaults to 0.05.
 
         Returns:
-            DataFrame: Disease target evidence
+            DataFrame | None: Disease target evidence
         """
         datasource_id = "gwas_credible_sets"
         datatype_id = "genetic_association"
 
+        # A set of optional columns need to be in the input datasets:
+        for c in ["diseaseIds", "pubmedId"]:
+            if c not in study_index.df.columns:
+                return None
```

Comment:
Why would you return `None` instead of raising an error?
I think I would check that the output contains the columns, not the input. What do you think?